content/zh-cn/designs/command/_index.md

```diff
@@ -0,0 +1,110 @@
+---
+title: "命令"
+weight: 11
+draft: true
+---
+<!-- {{< synopsis-command >}}
+-->
+
+可靠地要求设备执行单个操作并确认状态的一个请求实体。
+<!--more-->
+
+## 挑战
+
+IoT解决方案应该能以这样的方式与设备进行交互：解决方案或是使用该解决方案的人，可以可靠地要求设备执行某个动作。更进一步的，这样的交互需要在会出现间隙性中断的网络环境中进行，并通常需要使用资源非常有限的设备。
+
+## 解决方案
+
+IoT解决方案使用"命令"设计来要求设备执行操作，并通过利用一个简单的概念确保可靠的交互：除非被确认为成功，否则任何请求的操作都不会被视为成功。
+
+下图中显示的"命令"设计可以提供此功能：
+
+![Command Design](command.png) 
+([PPTx](atlas-command.pptx))
```

Comment:
This is no longer how the designs should link to PPTx downloads. Instead you can just link to `https://github.com/aws/iot-atlas/blob/master/content/en-us/designs/iot-atlas-patterns.pptx` using `{{< ref path="/designs/iot-atlas-patterns.pptx" lang="en" >}}`. That single PPTx in the English language directory contains all the diagrams of the site's Designs.

Reply:
The link is changed to  ([PPTx](/designs/iot-atlas-patterns.pptx)) .